src/index.html

```diff
@@ -22,6 +22,378 @@
+          BUY
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__description">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p>Product code: 195434</p>
+        </div>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <div>Price:</div>
+          <div>$2,199</div>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__description">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p>Product code: 195434</p>
+        </div>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <div>Price:</div>
+          <div>$2,199</div>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__description">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p>Product code: 195434</p>
+        </div>
+        <div class="card__stars">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <div>Price:</div>
+          <div>$2,199</div>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
All product cards contain the same information and image. If the intention is to display different products, ensure each card has unique content, such as different images, titles, product codes, and prices.